postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -22,6 +23,8 @@
 
 VALID_EXPLAIN_STATEMENTS = frozenset({'select', 'table', 'delete', 'insert', 'replace', 'update'})
 
+unix_epoch = datetime.datetime(1970, 1, 1, tzinfo=datetime.timezone.utc)
```

Comment:
I don't think we can always assume UTC here. Won't this depend on the configured TZ of the database?